DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/Processors/Factories/DicomProcessorFactory.cs

```diff
@@ -22,8 +26,23 @@ public IAnonymizerProcessor CreateProcessor(string method, JObject settingObject
                 "remove" => new RemoveProcessor(),
                 "refreshuid" => new RefreshUIDProcessor(),
                 "keep" => new KeepProcessor(),
-                _ => null,
+                _ => CreateCustomProcessor(method)
             };
         }
+
+        public void AddCustomerProcessor(string method, IAnonymizerProcessor processor)
```

Comment:
what's the relationship between create and add processor?

Reply:
AddCustomerProcessor() is a public API for user to add processor.
CreateCustomerProcessor() is a private function used for CreateProcessor() function.

I removed CreateCustomerProcessor() here.